back/src/bsdasris/resolvers/mutations/updateBsdasri.ts

```diff
@@ -78,16 +79,18 @@ const getFieldsAllorwedForUpdate = (bsdasri: Bsdasri) => {
   };
   return allowedFields[bsdasri.status];
 };
-const getRegroupedBsdasriArgs = inputRegroupedBsdasris => {
-  if (inputRegroupedBsdasris === null) {
-    return { set: [] };
+
+const getRegroupedBsdasriArgs = (
+  inputRegroupedBsdasris: RegroupedBsdasriInput[] | null | undefined
+) => {
+  if (inputRegroupedBsdasris === null || inputRegroupedBsdasris?.length === 0) {
```

Comment:
D'après la signature de la fonction `inputRegroupedBsdasris` peut être `null` ou `undefined`, y'a un `=` en trop du coup :

```suggestion
  if (inputRegroupedBsdasris == null || inputRegroupedBsdasris?.length === 0) {
```

Reply:
Si le le champ n'est pas passé (parce qu'on ne veut pas le modifier) , il est undefined, et dans ce cas je ne veux pas  dégrouper les bordereaux précédemment associés.